mailpoet/assets/css/src/components-automation-listing/_sections.scss

```diff
@@ -0,0 +1,171 @@
+:root {
+  --menu-width: 0;
+
+  @media screen and (min-width: 783px) {
+    --menu-width: 36px;
+  }
+
+  @media screen and (min-width: 961px) {
+    --menu-width: 160px;
+  }
+}
+
+@mixin full-width {
+  margin-left: calc(-50vw + var(--menu-width) / 2 + 50%);
+  padding-left: 104px;
+  padding-right: 104px;
+  width: calc(100vw - var(--menu-width));
+}
+
+.mailpoet-automation-section {
+  h2 {
+    font-size: 23px;
+    font-weight: 400;
+    line-height: 32px;
+    margin: 0;
+    padding: 0 0 8px;
+  }
+
+  p {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    margin: 0;
+    padding: 0 0 40px;
+  }
+}
+
+.mailpoet-automation-section-hero {
+  background: #fff;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-top: -20px;
+  padding: 65px;
+
+  @include full-width;
+
+  .mailpoet-automation-preheading {
+    display: block;
+    font-size: 11px;
+    letter-spacing: .2px;
+    line-height: 16px;
+    margin-bottom: 65px;
+    text-transform: uppercase;
+  }
+
+  h1 {
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 40px;
+  }
+
+  p {
+    font-size: 14px;
+    line-height: 22px;
+    margin-bottom: 32px;
+  }
+
+  > div {
+    width: 400px;
+  }
+}
+
+.mailpoet-section-templates {
+  padding: 48px 84px;
+
+  .mailpoet-section-template-list {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 40px;
```

Comment:
The boxes can have different heights now, I think it would be nicer to make them same-height (e.g. by CSS grid).

<img width="1472" alt="Screenshot 2022-10-27 at 9 35 00" src="https://user-images.githubusercontent.com/141436/198220007-144299fb-5447-4624-8ba9-b224e4961c7b.png">

Reply:
👍 and also always have some space between the boxes. Now on smaller displays, they are squashed together.

Reply:
Ensure space between: 8b8811e58
Same height: 50ff58b45

Reply:
After rebase:
space between: [c543b1a](https://github.com/mailpoet/mailpoet/pull/4461/commits/c543b1a313992a1c10fc42fe26b67bd01cb5e784)
same height: [496d616](https://github.com/mailpoet/mailpoet/pull/4461/commits/496d6162668e312fb89b6240fd758067addc3b25)

Reply:
Good catch. Thank you!

4bb6330a5 and 9b6a0db01 ensure now same width and top alignment.